csi.proto

```diff
@@ -1296,6 +1357,31 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+// VolumeCondition represents the current health condition of a volume
+message VolumeCondition {
+  option (alpha_message) = true;
+
+  // Status represents the current status of a condition
+  message Status {
+    enum Type {
+      CONDITION_UNKNOWN = 0;
+      CONDITION_TRUE = 1;
+      CONDITION_FALSE = 2;
+    }
+    Type type = 1;
+  }
+
+  oneof type {
+    // Indicates the health condition of a volume is not normal.
+    // This field is REQUIRED.
+    Status abnormal = 1;
```

Comment:
We probably should use https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.BoolValue here if the intention is just true/false/unset. We use the same in `ProbeResponse`

Reply:
Changed to BoolValue.